### PR TITLE
New primary/secondary colors for improved color contrast

### PR DESCRIPTION
**Pull request by agessler**
@apepper unglier but better contrast - I could try other combinations to improve visual design. Wave test is now happy on every page.

**OLD**
```
--bs-primary: #274486
--bs-secondary: #39a9eb
--bs-body-color:  #677788;
```

**NEW**
```
--bs-primary: #072b46; 
--bs-secondary: #0e5588; 
--bs-body-color: #4f557a;
```

### CODE REVIEW

File: src/assets/stylesheets/index.scss [hunk 1]
```diff
@@ -19,20 +19,20 @@
   --border: rgba(0, 0, 0, 0.1);
 
   // The variable var(--bs-primary) is utilized in packages like `scrivito-neoletter-form-widgets`.
-  --bs-primary: #274486;
+  --bs-primary: #072b46; // #274486 before
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:22]: Please adjust `SiteColorsPicker.tsx` with the new default color:

```tsx
color={page.get('siteColorPrimary') || '#072b46'}
```

File: src/assets/stylesheets/index.scss [hunk 2]
```diff
@@ -19,20 +19,20 @@
...
 
   // The variable var(--bs-secondary) is utilized in packages like `scrivito-neoletter-form-widgets`.
-  --bs-secondary: #39a9eb;
+  --bs-secondary: #0e5588; // #39a9eb before
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:25]: Same here, please adjust `SiteColorsPicker.tsx` with:

```tsx
color={page.get('siteColorSecondary') || '#0e5588'}
```

File: src/assets/stylesheets/index.scss (hunk 1 above)
Comment (apepper) [inline src/assets/stylesheets/index.scss:22]: Please remove the `// #274486 before` comment. It's no longer needed.

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (apepper) [inline src/assets/stylesheets/index.scss:25]: Same here: No `// #39a9eb before` comment.

File: src/assets/stylesheets/index.scss [hunk 3]
```diff
@@ -19,20 +19,20 @@
...
-  --bs-primary-darken: #1e3466;
-  --bs-secondary-lighten: #5eb9ef;
-  --bs-secondary-darken: #1798e4;
+  --bs-primary-lighten: #063d66;
+  --bs-primary-darken: #071d2d;
+  --bs-secondary-lighten: #0b5f9b;
+  --bs-secondary-darken: #0d4166;
 
   --bs-success: #21704b;
   --bs-info: #39a9eb;
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:38]: `--bs-info` is still set to the "old" `bs-secondary` (`#39a9eb`). Should it be set to the new `bs-secondary` (`#0e5588`) as well?

File: src/assets/stylesheets/index.scss [hunk 4]
```diff
@@ -19,20 +19,20 @@
...
   // --bs-secondary-lighten: hsl(from var(--bs-secondary) h s calc(l + 8));
   // --bs-secondary-darken: hsl(from var(--bs-secondary) h s calc(l - 8));
-  --bs-primary-lighten: #3054a6;
-  --bs-primary-darken: #1e3466;
-  --bs-secondary-lighten: #5eb9ef;
-  --bs-secondary-darken: #1798e4;
+  --bs-primary-lighten: #063d66;
+  --bs-primary-darken: #071d2d;
+  --bs-secondary-lighten: #0b5f9b;
+  --bs-secondary-darken: #0d4166;
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:35]: If I "lighten" and "darken" the values using the `SiteColorsPicker` I receive the following values:

```scss
--bs-primary: #072b46;
--bs-primary-lighten: #0B426B;
--bs-primary-darken: #031421;
--bs-secondary: #0e5588;
--bs-secondary-lighten: #126CAD;
--bs-secondary-darken: #0A3E63;
```

Ideally they should result in the same color value. Either adjust the `index.scss` values or adjust the formular for `SiteColorsPicker` to calculate lighten/darken correctly. Our previous plan was to eventually use this:

```scss
--bs-primary-lighten: hsl(from var(--bs-primary) h s calc(l + 8));
--bs-primary-darken: hsl(from var(--bs-primary) h s calc(l - 8));
--bs-secondary-lighten: hsl(from var(--bs-secondary) h s calc(l + 8));
--bs-secondary-darken: hsl(from var(--bs-secondary) h s calc(l - 8));
```

**Review by apepper — CHANGES_REQUESTED** [review]
FEEDBACK ([575660b](https://github.com/Scrivito/scrivito-portal-app/pull/632/commits/575660bb5b936d01a04a83bee9d636b213cdf4ee))

Really nice 🚀!

All pages now have a 100% correct color contrast 💯!

Please see my comments for details, that still need some work.

File: src/assets/stylesheets/index.scss (hunk 3 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss:38]: Var handover would be possible but secondary can be a green or red.. so I just copied the secondary.

File: src/assets/stylesheets/index.scss (hunk 4 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss]: Done with latest commit

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss]: Done with latest commit

File: src/assets/stylesheets/index.scss (hunk 1 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss]: Done with latest commit

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss]: Done with latest commit

File: src/assets/stylesheets/index.scss (hunk 1 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss]: Done with latest commit

**Review by apepper — APPROVED** [review]
LGTM ([d3f84d9](https://github.com/Scrivito/scrivito-portal-app/pull/632/commits/d3f84d9cc5200c761206d4424dc0cbdef0e8054e))

Nice 🚀!

**Comment by apepper** [comment]
The undisputed stuff of this PR has been merged with https://github.com/Scrivito/scrivito-portal-app/pull/648. Closing for now.

**Pull request closed without merging**